interface/wx/thread.h

```diff
@@ -503,10 +503,25 @@ class wxThreadHelper
 
         @since 2.9.2
 
-        @see OnDelete()
+        @see OnDelete(), OnExit()
     */
     virtual void OnKill();
 
+    /**
+        Callback called by Exit() before actually exiting the thread.
+        This function will not be called if the thread was @ref Kill() killed.
+
+        This function can be overridden by the derived class to perform some
+        specific task when the thread is exited. Notice that it will be
+        executed in the context of the thread that called Exit() and <b>not</b>
```

Comment:
I don't think this sentence is accurate, as `Exit()` itself can only be called from the thread context. So in this case `OnExit()` will also be called from the thread context and I don't think it can ever be called in another thread context.

Reply:
Removed.